src/queries/duckdb/admin_boundaries.sql

```diff
@@ -1,19 +1,20 @@
 LOAD httpfs;
 LOAD spatial;
+SET s3_region='us-west-2';
 
 CREATE VIEW admins_view AS (
     SELECT
         *
     FROM
-        read_parquet('s3://overturemaps-us-west-2/release/__OVERTURE_RELEASE/theme=admins/type=*/*', filename=true, hive_partitioning=1)
+        read_parquet('s3://overturemaps-us-west-2/release/2024-02-15-alpha.0/theme=admins/type=*/*', filename=true, hive_partitioning=1)
```

Comment:
_Nice catch_ on this one — we'll need to remember that some of these queries are _very_ version specific and be sure not to generalize for them.

Reply:
Agreed for now. But I would like to move toward generalize as many example queries as possible over the next few releases.